mk/spksrc.service.installer.functions

```diff
@@ -68,15 +68,16 @@ initialize_variables ()
 
 install_python_virtualenv ()
 {
-    # Output Python version (confirming PATH)
+    # Print out default python version found in PATH
     python3 --version
 
+    # Print out default pip version
+    read -r -a PIP_VERSION 2>/dev/null <<< $(python3 -m pip --version)
+    echo "INFO: Default pip version found ${PIP_VERSION[1]}"
+
```

Comment:
@mreid-tt (/cc @th0ma7) This change breaks the installer on all SRM and DSM 5.2 systems

This file must be `/bin/sh` (i.e. ash) compatible.
At least `${PIP_VERSION[1]}` is valid for `/bin/bash` only and breaks the installer in `/bin/sh` environments for all kind of packages - not only packages installing a python virtualenv.

**Please regard lines 5-6 at the top of this file.**

Reply:
The culprit is most probably:
```
read -r -a PIP_VERSION 2>/dev/null <<< $(python3 -m pip --version)
```
A potential fix would be:
```
read -r -a PIP_VERSION 2>/dev/null << EOF
$(python3 -m pip --version)
EOF
```

Reply:
Resolved by #6028.